gradle.properties

```diff
@@ -1,16 +1,18 @@
 # Done to increase the memory available to gradle.
 org.gradle.jvmargs=-Xmx1G
+org.gradle.parallel=true
```

Comment:
Does this matter? To my knowledge, Yarn doesn't have subprojects. This flag only affects multiproject builds. (See [this discord discussion](https://discord.com/channels/507304429255393322/592749519171616796/1021440447144546384))

Reply:
No, I double checked and it doesnt, no real harm in setting it tho.